src/lib/temple/front/assets.ts

```diff
@@ -220,11 +220,16 @@ export const useAvailableRoute3Tokens = () => {
 
 function makeAssetsSortPredicate(balances: Record<string, BigNumber>, fiatToTokenRates: Record<string, string>) {
   return (tokenASlug: string, tokenBSlug: string) => {
-    if (tokenASlug === TEZ_TOKEN_SLUG) {
+    if (tokenASlug === TEZ_TOKEN_SLUG && tokenBSlug === TEMPLE_TOKEN_SLUG) {
       return -1;
     }
-
-    if (tokenBSlug === TEZ_TOKEN_SLUG) {
+    if (tokenBSlug === TEZ_TOKEN_SLUG && tokenASlug === TEMPLE_TOKEN_SLUG) {
+      return 1;
+    }
+    if (tokenASlug === TEZ_TOKEN_SLUG || tokenASlug === TEMPLE_TOKEN_SLUG) {
+      return -1;
+    }
+    if (tokenBSlug === TEZ_TOKEN_SLUG || tokenBSlug === TEMPLE_TOKEN_SLUG) {
       return 1;
     }
```

Comment:
Please refactor these logic using a list of slugs of the tokens that should be first